xml/System.Windows.Automation/NotificationEventArgs.xml

```diff
@@ -1,133 +1,133 @@
-      </ReturnValue>
-      <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
-      </Docs>
-    </Member>
-    <Member MemberName="NotificationProcessing">
-      <MemberSignature Language="C#" Value="public System.Windows.Automation.AutomationNotificationProcessing NotificationProcessing { get; }" />
-      <MemberSignature Language="ILAsm" Value=".property instance valuetype System.Windows.Automation.AutomationNotificationProcessing NotificationProcessing" />
-      <MemberSignature Language="DocId" Value="P:System.Windows.Automation.NotificationEventArgs.NotificationProcessing" />
-      <MemberSignature Language="VB.NET" Value="Public ReadOnly Property NotificationProcessing As AutomationNotificationProcessing" />
-      <MemberSignature Language="F#" Value="member this.NotificationProcessing : System.Windows.Automation.AutomationNotificationProcessing" Usage="System.Windows.Automation.NotificationEventArgs.NotificationProcessing" />
-      <MemberSignature Language="C++ CLI" Value="public:&#xA; property System::Windows::Automation::AutomationNotificationProcessing NotificationProcessing { System::Windows::Automation::AutomationNotificationProcessing get(); };" />
-      <MemberType>Property</MemberType>
-      <AssemblyInfo>
-        <AssemblyName>UIAutomationTypes</AssemblyName>
-        <AssemblyVersion>6.0.0.0</AssemblyVersion>
-      </AssemblyInfo>
-      <ReturnValue>
-        <ReturnType>System.Windows.Automation.AutomationNotificationProcessing</ReturnType>
-      </ReturnValue>
-      <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
-      </Docs>
-    </Member>
-  </Members>
-</Type>
+<Type Name="NotificationEventArgs" FullName="System.Windows.Automation.NotificationEventArgs">
+  <TypeSignature Language="C#" Value="public sealed class NotificationEventArgs : System.Windows.Automation.AutomationEventArgs" />
+  <TypeSignature Language="ILAsm" Value=".class public auto ansi sealed beforefieldinit NotificationEventArgs extends System.Windows.Automation.AutomationEventArgs" />
+  <TypeSignature Language="DocId" Value="T:System.Windows.Automation.NotificationEventArgs" />
+  <TypeSignature Language="VB.NET" Value="Public NotInheritable Class NotificationEventArgs&#xA;Inherits AutomationEventArgs" />
+  <TypeSignature Language="F#" Value="type NotificationEventArgs = class&#xA;    inherit AutomationEventArgs" />
+  <TypeSignature Language="C++ CLI" Value="public ref class NotificationEventArgs sealed : System::Windows::Automation::AutomationEventArgs" />
+  <AssemblyInfo>
+    <AssemblyName>UIAutomationTypes</AssemblyName>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Windows.Automation.AutomationEventArgs</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <summary>NotificationEventArgs event args class</summary>

```

Comment:
@ryalanms this isn't very helpful for the user. Can you please provide a better description?